client/dashboard/me/monetize-subscriptions/monetize-subscription.tsx

```diff
@@ -0,0 +1,310 @@
+		monetizeSubscriptionResumeAutoRenew( subscriptionId )
+	);
+
+	const isRenewable = subscription && ( subscription.renew_interval || subscription.is_renewable ); // can remove renew_interval once backend is deployed
+	const isAutoRenewing = isRenewable && subscription.renew_interval;
+	const isProduct = !! subscription && ! isRenewable;
+	const isDisabledAutorenewing = isRenewable && ! subscription.renew_interval;
+	const isUpdating = isEnablingAutoRenew || isDisablingAutoRenew;
+
+	return (
+		<PageLayout
+			size="small"
+			header={
+				<PageHeader
+					prefix={ <BackButton /> }
+					title={ isProduct ? __( 'Product Details' ) : __( 'Subscription Details' ) }
+				/>
+			}
+		>
+			{ isStoppingSubscription && (
+				<Notice status="info">
+					{ isProduct ? __( 'Removing this product' ) : __( 'Stopping this subscription' ) }
+				</Notice>
+			) }
+			{ isUpdating && <Notice status="info">{ __( 'Updating subscription auto-renew' ) }</Notice> }
+			{ subscription && (
+				<Card>
+					<MonetizeSiteHeader name={ subscription.site_title } domain={ subscription.site_url } />
+
+					<CardHeader>
+						<HStack alignment="center">
+							<VStack>
+								<HStack alignment="center">
+									<Text weight={ 700 } size="title" as="h3">
+										{ subscription.title }
+									</Text>
+									<Text weight={ 700 }>
+										{ formatCurrency(
+											parseFloat( subscription.renewal_price ),
+											subscription.currency
+										) }
+									</Text>
+								</HStack>
+
+								{ ! isProduct && (
+									<center>
+										{ subscription.renew_interval
+											? __( 'Auto-renew is ON' )
+											: __( 'Auto-renew is OFF' ) }
+									</center>
+								) }
+							</VStack>
+						</HStack>
+					</CardHeader>
+
+					<CardBody>
+						<HStack>
+							<Card>
+								<CardHeader>{ __( 'Renew interval' ) }</CardHeader>
+								<CardBody>
+									<HStack alignment="center">{ subscription.renew_interval || '-' }</HStack>
+								</CardBody>
+							</Card>
+							<Card>
+								<CardHeader>{ __( 'Subscribed On' ) }</CardHeader>
+								<CardBody>{ formatDate( subscription.start_date, 'MMM dd, yyyy' ) }</CardBody>
+							</Card>
+							<Card>
+								<CardHeader>
+									{ isDisabledAutorenewing ? __( 'Expires on' ) : __( 'Renews on' ) }
+								</CardHeader>
+
+								<CardBody>
+									{ subscription.end_date
+										? formatDate( subscription.end_date, 'MMM dd, yyyy' )
+										: __( 'Never Expires' ) }
+								</CardBody>
+							</Card>
+						</HStack>
+					</CardBody>
+					<ActionList>
+						{ isRenewable && (
+							<AutoRenewButton
+								subscription={ subscription }
+								isAutoRenewing={ isAutoRenewing }
+								isUpdating={ isUpdating }
+								disableAutoRenew={ disableAutoRenew }
+								enableAutoRenew={ enableAutoRenew }
+								isProduct={ isProduct }
+							/>
+						) }
+
+						<StopSubscriptionButton
```

Comment:
Do we need to repeat that copy 3 times? What about only showing the CTA Button instead?

<img width="768" height="545" alt="Image" src="https://github.com/user-attachments/assets/07eb434f-e927-408b-bf81-ac1a32f7d0eb" />

Reply:
Also on the removal experience, I see some issues:
- We're not asking if the user really wants to do that (which is a pattern we expect to see on HD. See p1755089977975119-slack-C0982082MSR). 
- Once I click on Remove, I'd expect the button to be blocked and have a loader on it
- Showing a notice on the top with the "Removing this product" doesn't feel the right pattern, especially since it makes the screen jump:

https://github.com/user-attachments/assets/05f0df39-f477-4e21-a3ab-51b8d7c80a64